gno.land/cmd/gnoland/config_get_test.go

```diff
@@ -616,27 +608,19 @@ func TestConfig_Get_P2P(t *testing.T) {
 			},
 			true,
 		},
-		{
-			"upnp toggle",
-			"p2p.upnp",
-			func(loadedCfg *config.Config, value []byte) {
-				assert.Equal(t, loadedCfg.P2P.UPNP, unmarshalJSONCommon[bool](t, value))
-			},
-			false,
-		},
 		{
 			"max inbound peers",
 			"p2p.max_num_inbound_peers",
 			func(loadedCfg *config.Config, value []byte) {
-				assert.Equal(t, loadedCfg.P2P.MaxNumInboundPeers, unmarshalJSONCommon[int](t, value))
+				assert.Equal(t, loadedCfg.P2P.MaxNumInboundPeers, unmarshalJSONCommon[uint64](t, value))
```

Comment:
things that you often do arithmetic operations on especially comparison operations, should remain as signed numbers.

x > y is often expressed as x - y > 0, which doesn't work with unsigned numbers, more bug prone. we are exchanging 1 bit for code brittleness.